xcp_d/workflows/base.py

```diff
@@ -749,26 +760,32 @@ def init_subject_wf(
                     ("outputnode.t1w", "inputnode.t1w"),
                     ("outputnode.t2w", "inputnode.t2w"),
                 ]),
-                (load_atlases_wf, postprocess_bold_wf, [
-                    ("outputnode.atlas_names", "inputnode.atlas_names"),
-                    ("outputnode.atlas_files", "inputnode.atlas_files"),
-                    ("outputnode.atlas_labels_files", "inputnode.atlas_labels_files"),
-                ]),
             ])
             # fmt:on
 
-            if cifti:
+            if atlases:
                 # fmt:off
                 workflow.connect([
                     (load_atlases_wf, postprocess_bold_wf, [
-                        (
-                            "outputnode.parcellated_atlas_files",
-                            "inputnode.parcellated_atlas_files",
-                        ),
+                        ("outputnode.atlas_files", "inputnode.atlas_files"),
+                        ("outputnode.atlas_labels_files", "inputnode.atlas_labels_files"),
                     ]),
                 ])
                 # fmt:on
-            else:
+
+                if cifti:
```

Comment:
Does this need to be reachable if there are no atlases?

Reply:
No, it shouldn't be reachable, since it's passing atlas files that should only be used if parcellation is done.